bin/create-docs-pr.js

```diff
@@ -0,0 +1,303 @@
+/*
+ * Copyright 2022 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+
+const fs = require('fs')
+const path = require('path')
+const { program } = require('commander')
+
+const Github = require('./github')
+const git = require('./git-commands')
+
+const DEFAULT_FILE_NAME = 'NEWS.md'
+/** e.g. v7.2.1 */
+const TAG_VALID_REGEX = /v\d+\.\d+\.\d+/
+
+program.requiredOption('--tag <tag>', 'tag name to create GitHub release for')
+program.option('--remote <remote>', 'remote to push branch to', 'origin')
+program.option('--username <github username>', 'Owner of the fork of docs-website')
+program.option(
+  '--changelog <changelog>',
+  'Name of changelog(defaults to NEWS.md)',
+  DEFAULT_FILE_NAME
+)
+program.option('-f --force', 'bypass validation')
+program.option('--dry-run', 'executes script but does not commit nor create PR')
+program.option(
+  '--repo-path <path',
+  'Path to the docs-website fork on local machine',
+  'docs-website'
+)
+const RELEASE_NOTES_PATH =
+  './src/content/docs/release-notes/agent-release-notes/nodejs-release-notes'
+
+async function createRelease() {
```

Comment:
Would prob rename this function to be something like `createReleaseNotePr` or something more specific to the task.